tests/fips/strongswan/strongswan_client.pm

```diff
@@ -0,0 +1,79 @@
+# strongswan test
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Package: strongswan
+# Summary: FIPS: strongswan_client
+#          In fips mode, testing strongswan
+#
+# Maintainer: Liu Xiaojing <xiaojing.liu@suse.com>
+# Tags: poo#108620
```

Comment:
Please add tc#$tc-num since there is one.

Reply:
added